glommio/src/executor/mod.rs

```diff
@@ -2546,22 +2749,22 @@ mod test {
             let tq2_count = Rc::new(RefCell::new(vec![0, 0]));
             let now = Instant::now();
 
-            let t1 = Local::local_into(
+            let t1 = crate::local_into(
```

Comment:
I named the free function `spawn_` to make it clearer what it does. But looking back at it it's just confusing. Would you prefer both to be called `spawn_local_*` or `local_*`?

Reply:
I kinda like spawn_local.. I think there's strong precedence for this because of tokio and it's 'spawn' method. I know it's longer but if this is about discoverability, that's the solution I think makes the most sense. I guess we could debate that it could just be 'spawn' but it might be unclear that we are always single-threaded so I think spawn_local is the simplest.

Reply:
All the functions that spawn now begin with `spawn_*`. Thanks!